invenio_assets/assets/package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "invenio-assets",
-  "version": "1.0.0",
+  "version": "1.3.1",
```

Comment:
Nice to have, but not really needed, given that it is not published on NpmJS

Reply:
Oh that was not the point to do that. It was really irritating to see the wrong version in the log outputs.